src/index.js

```diff
@@ -11,9 +11,9 @@ export default class SomethingNice extends PureComponent {
 
   render() {
     return (
-      <span>
-        { this.message }
+      <span style={this.props.style} className={this.props.className}>
```

Comment:
Mind adding these to the prop types, please?

Reply:
I hope i made it right

Reply:
Hi @medonagy45 . Please look at https://github.com/xnt/react-something-nice/blob/0aa122402bb507a00ffc0ac50f458cb72c0e97df/src/index.js#L29 .

`PropTypes` is a validator that can help us check if a property is passed, its type, its shape, and sometimes even helps a bit with our IDE's autocompletion and some linting tools (i.e. checking for typos).

Here is some doc on PropTypes - https://reactjs.org/docs/typechecking-with-proptypes.html

Let me know if you think you can implement the checks for `style` and `className`. What you've done so far is great! So I have no problem with merging your change and then implementing that validation. That being said, I wouldn't like to steal the spotlight from you unless you're cool with it.

Thank you *so much* for your contributions!

Reply:
Hi @xnt ,
I have made the changes , check it  and feed me back